docs/otlp/getting-started-console/Program.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+public class Program
+{
+    private static readonly ActivitySource MyActivitySource = new("MyCompany.MyProduct.MyLibrary");
+    private static readonly Meter MyMeter = new("MyCompany.MyProduct.MyLibrary");
+    private static readonly Counter<int> MyCounter = MyMeter.CreateCounter<int>("execution.count");
+
+    public static void Main()
+    {
+        // Note: OpenTelemetrySdk.Create was added in 1.10.0
```

Comment:
This won't work for people until they can get access to `1.10.0`. When will that ship?

Reply:
Nov this year.

Reply:
We could put out an alpha if you think that would help? I would prefer to not use the new style yet in guides but you can't use the cross-cutting "UseOtlpExporter" call with the old style. I felt that was more important for these particular guides.

Reply:
I guess it gets to the question of "who do you expect to read/follow these instructions between now and when `1.10.0` ships?"

If it is someone who you expect to know how to get this code to work (which excludes me), then the current form is fine.

But if it is someone who is just getting started here, then maybe we need to update the instructions to tell them how to make this app work. Or change the app to make it just work for right now. As it stands, I don't believe a new person would be successful trying to follow these instructions since this code doesn't compile.

Reply:
I put some stronger notices in there that it won't work/is a work in progress. But I'm hoping it is short lived and I can get it working as soon as we drop an alpha/beta.